src/Model/Model.ts

```diff
@@ -477,6 +478,21 @@ export class Model<T extends IModelData> implements IModel<T> {
     }
   }
 
+  /**
+   * Get the DataService associated to the relationship specified at the given name of the related field
+   *
+   * @param {string} relationshipKey
+   * @returns {DataService<any>}
```

Comment:
Same thing here.  Should we set a type for this?

Reply:
See above